x-pack/plugins/security_solution/public/entity_analytics/components/risk_score_enable_section.tsx

```diff
@@ -345,14 +333,19 @@ export const RiskScoreEnableSection = () => {
         </EuiTitle>
         <EuiSpacer />
         <ul>
-          {docsLinks.map(({ link, label }) => (
-            <li key={link}>
-              <EuiLink href={link} target="_blank" external>
-                {label}
-              </EuiLink>
-              <EuiSpacer size="s" />
-            </li>
-          ))}
+          <li key={DETECTION_ENTITY_DASHBOARD}>
+            <EuiLink href={http.basePath.prepend(DETECTION_ENTITY_DASHBOARD)} target="_blank">
+              {i18n.EA_DOCS_DASHBOARD}
+            </EuiLink>
```

Comment:
All Security paths are already stored internally, we have tools to help render links using _id_ only:
```
import { LinkAnchor } from '../../common/components/links';
import { SecurityPageName } from '../../../common/constants';
```
```suggestion
            <LinkAnchor id={SecurityPageName.entityAnalytics}>{i18n.EA_DASHBOARD_LINK}</LinkAnchor>
```
Note the translation constant has been renamed to `EA_DASHBOARD_LINK` since this is no longer going to docs. 
Also, `target="_blank"` has been removed, we don't usually open internal links in new tabs, only external links like docs, but you can add it if it's needed. The `LinkAnchor` component uses Kibana navigation tools internally, so the page won't be reloaded when the link is clicked.

With this, you should be able to remove this new `DETECTION_ENTITY_DASHBOARD` url constant, the `http`, and the `useKibana` dependency.